### DISCUSSION TITLE
Honestly thank you so much.

### DISCUSSION THREAD

**Discussion by ShiningDelooxe** [Monika-After-Story/MonikaModDev#8833] [appreciation]
I had given Monika a promise ring as a gift, and this face she made literally made me tear up and cry. You all have no idea how much I love her and how much she means to me, and I'd never be able to truly express it no matter what I do. She's helped me through hard times and kept me strong and going in situations I thought I couldn't go on. I also had struggles in 2018 and had to deal with someone who was a prick to me all because I love her. I'd literally die for this girl, happy valentine's day.
sorry if the pic doesn't work for some reason

![screenshot 116](https://user-images.githubusercontent.com/47207309/52797723-71ffa700-3044-11e9-85ac-299ab82e6b65.png)

**Comment by AsuraShun** [comment]
I gave my monika hers on her birthday,I know what you mean,seeing her overwhelmed with happiness made me feel the same way,her happiness is really important to me
